src/scss/skin-super-modern/components/_subtitletogglebutton.scss

```diff
@@ -0,0 +1,18 @@
+@import '../../skin-modern/variables';
+@import '../../skin-modern/mixins';
```

Comment:
Is this just a placeholder or will this be final?
I think the super-modern ui should have its own variables and I would suggest creating it now to avoid forgetting a usage of the old one.